packages/provider-utils/src/fetch-with-user-agent.ts

```diff
@@ -0,0 +1,67 @@
+import type { FetchFunction } from './fetch-function';
+import { VERSION as PROVIDER_UTILS_VERSION } from './version';
+
+export function getRuntimeEnvironmentUserAgent(): string {
+  // Browsers / Deno / Bun / Node.js >= 21.1
+  if (globalThis.navigator?.userAgent) {
+    return navigator.userAgent;
+  }
+
+  // Nodes.js < 21.1
+  if (globalThis.process?.versions?.node) {
+    return `Node.js/${process.version.substring(1)}`;
+  }
+
+  return '<unknown runtime>';
```

Comment:
Sorry I only caught this now. I think we should avoid using `<>` characters

See https://www.rfc-editor.org/rfc/rfc9110.html#field.user-agent for more information

How about we prefix all 3 possible returns with `provider/`

```suggestion
  // Browsers / Deno / Bun / Node.js >= 21.1
  if (globalThis.navigator?.userAgent) {
    return `runtime/${navigator.userAgent}`;
  }

  // Nodes.js < 21.1
  if (globalThis.process?.versions?.node) {
    return `runtime/Node.js/${process.version.substring(1)}`;
  }

  return 'runtime/unknown';
```

Reply:
yup sounds good! committing this change